lib/config.ts

```diff
@@ -88,6 +88,13 @@ export interface Config {
    */
   webDriverProxy?: string;
 
+  /**
+   * If specified, connect to webdriver through a proxy that manages client-side
+   * synchronization. This option is intended for testing an  experimental feature
+   * and is not meant to be used by end users.
```

Comment:
This is a bit scary - maybe just 

"Blocking Proxy is an experimental feature and may change without notice."?

Also, I think this should be boolean?